docs/styleguide/templates/master.html

```diff
@@ -2,12 +2,16 @@
 <html lang="en-us">
   <head>
 {% include "app_meta.html" %}
-	<link rel="stylesheet" href="css/techne-{% if debug %}debug{% else %}all{% endif %}{% if env == "production" %}.min{% endif %}.css">
+	<link rel="stylesheet" href="css/techne-{% if debug %}debug{% else %}all{% endif %}{% if env == 'production' %}.min{% endif %}.css">
 
     <link rel="stylesheet" href="css/techne-styleguide.css">
 {% block css %}
 
 {% endblock %}
+      <script src="js/techne-{% if debug %}debug{% else %}all{% endif %}{% if env == 'production' %}.min{% endif %}.js"></script>
```

Comment:
A few things ...

1. Since this JS is specific to the style guide and not the library, let's call it `techne-styleguide.js` to match the name of the styleguide CSS.
2. All of this should go at the bottom of the HTML page, before the closing `<body>` tag.
3. The debug logic can be removed since there is no debug JS.